gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -363,6 +363,10 @@ MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_Ver
 
 	buffer->formatAndOutput(env, 1, "<system>");
 	buffer->formatAndOutput(env, 2, "<attribute name=\"physicalMemory\" value=\"%llu\" />", omrsysinfo_get_physical_memory());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"addressablePhysicalMemory\" value=\"%llu\" />", omrsysinfo_get_addressable_physical_memory());
+	buffer->formatAndOutput(
+		env, 2, "<attribute name=\"using container and its memory limit is set\" value=\"%s\" />",
```

Comment:
Do this as well: https://github.com/eclipse/omr/pull/7199#discussion_r1422921618

In this change that flag would default to false, but you'd need to follow up with another change where some other code sets the value. Specifically for OpenJ9 you'd need a change there (OMR GC does not look into container limit).

Also I prefer a bit shorter name like 'container memory limit set/selected'. The fact that the container limit is set implies that we are using container.